libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -199,6 +199,18 @@ export const NavbarLogoWrapper = styled('div')`
 
 export const NavbarSpacer = styled('div')``
 
+const OpenInvoiceButton = styled(Button)`
+  &::after {
+    content: 'Rechnung';
```

Comment:
Shouldn't be done with before and after as it's an accessibility issue.
Use https://mui.com/system/display/